docs/dev/table/sqlClient.md

```diff
@@ -459,6 +466,50 @@ Web interface: http://localhost:8081
 
 {% top %}
 
+SQL Views
+---------
+
+Views allow to define virtual tables from SQL queries. The view definition is parsed and validated immediately. However, the actual execution happens when the view is accessed during the submission of a general `INSERT INTO` or `SELECT` statement.
+
+Views can either be defined in [environment files](sqlClient.html#environment-files) or within the CLI session.
+
+The following example shows how to define multiple views in a file:
+
+{% highlight yaml %}
+views:
+  - name: MyRestrictedView
+    query: "SELECT MyField2 FROM MyTableSource"
+  - name: MyComplexView
+    query: >
+      SELECT MyField2 + 42, CAST(MyField1 AS VARCHAR)
+      FROM MyTableSource
+      WHERE MyField2 > 200
+{% endhighlight %}
+
+Similar to table sources and sinks, views defined in a session environment file have highest precendence.
+
+Views can also be created within a CLI session using the `CREATE VIEW` statement:
+
+{% highlight text %}
+CREATE VIEW MyNewView AS SELECT MyField2 FROM MyTableSource
+{% endhighlight %}
+
+The `SHOW VIEW` statement allows for printing a previously created view again:
```

Comment:
OK, I don't mind listing views are tables in `SHOW TABLES`.

However, if there is no way to distinguish a view from a table, we can also remove the `SHOW VIEW xxx` command. Why should somebody try to look up the definition of a view if he's not aware that it is a view. He can only know that if he created the view himself and in that case probably know the definition.

I also think the syntax of `SHOW VIEW xxx` is not well chosen because it is inconsistent with `SHOW TABLES` and `SHOW FUNCTIONS`. It gives *details about a single view* where as the other `SHOW` commands *list all elements of a type*.  I think the semantics are more similar to the `DESCRIBE` command. We could rename `SHOW VIEW` to ``DESCRIBE VIEW`.

Reply:
FYI: In MySQL it is called [SHOW CREATE VIEW](https://dev.mysql.com/doc/refman/8.0/en/show-create-view.html) and many vendors don't even offer this functionality which I find weird (e.g. Snowflake only shows the schema for [DESCRIBE VIEW](https://docs.snowflake.net/manuals/sql-reference/sql/desc-view.html)).

I'm fine with removing `SHOW VIEW` for now?

Reply:
OK, let's remove it for now and figure that out later.